### PR TITLE
parser:add support for creating view -1

### PR DESCRIPTION
**Pull request by tuichenchuxin**
For #61
- parser: support for creating view - 1

### CODE REVIEW

File: pisa-proxy/parser/mysql/src/grammar.y [hunk 1]
```diff
@@ -2372,7 +2375,7 @@ opt_expr_list -> Vec<Expr>:
   ;
 
 expr_list -> Vec<Expr>:
-    expr
+    expr %prec LOWER_THEN_COMMA
```
Comment (xuanyuan300) [inline pisa-proxy/parser/mysql/src/grammar.y:2378]: Is there conflict here?

File: pisa-proxy/parser/mysql/src/grammar.y [hunk 2]
```diff
@@ -6262,6 +6270,290 @@ default_encryption -> CreateDatabaseOption:
...
+        }
+        ;
+
+sp_c_chistic -> String:
+          sp_chistic            { $1 }
+        | 'DETERMINISTIC'     { String::from("DETERMINISTIC") }
+        | not 'DETERMINISTIC' { String::from("NOT DETERMINISTIC") }
+        ;
+
+sp_chistic -> String:
```
Comment (xuanyuan300) [inline pisa-proxy/parser/mysql/src/grammar.y:6526]: This rule should also return a `Enum` type.

File: pisa-proxy/parser/mysql/src/grammar.y [hunk 3]
```diff
@@ -6262,6 +6270,290 @@ default_encryption -> CreateDatabaseOption:
...
+          { String::from("CONTAINS SQL") }
+        | 'READS' 'SQL' 'DATA'
+          { String::from("READS SQL DATA") }
+        | 'MODIFIES' 'SQL' 'DATA'
+          { String::from("MODIFIES SQL DATA") }
+        | sp_suid
+          { $1 }
+        ;
+
+sp_suid -> String:
```
Comment (xuanyuan300) [inline pisa-proxy/parser/mysql/src/grammar.y:6547]: This rule should also return a Enum type.

File: pisa-proxy/parser/mysql/src/grammar.y (hunk 2 above)
Comment (tuichenchuxin) [reply pisa-proxy/parser/mysql/src/grammar.y]: Ok, Thank you , Done.

File: pisa-proxy/parser/mysql/src/grammar.y (hunk 1 above)
Comment (xuanyuan300) [reply pisa-proxy/parser/mysql/src/grammar.y:2379]: Ok. Currently `Shift(",") / Reduce(expr_list: "expr")`  conflict  has not reloved yet.

**Review by xuanyuan300 — APPROVED** [review]

**Review by wbtlb — APPROVED** [review]

**Pull request merged**